front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -259,5 +245,27 @@ export const sortSelectedModeCurves = (curvesList: ConditionalEffortCurve[]) =>
   return allConds.concat(onlyPR, onlyEP, nulls);
 };
 
-export const isElectric = (rollingStock: RollingStock) =>
-  some(Object.values(rollingStock.effort_curves.modes), (effortCurve) => effortCurve.is_electric);
+export function makeTractionMode(
```

Comment:
I don't really like the name of this function, do you think this function is really necessary ?

Reply:
renamed to makeEffortCurve. It’s needed for the tests